rasa_addons/core/nlg/bftemplate.py

```diff
@@ -85,11 +89,31 @@ async def generate(
             fallback_language=fallback_language,
         )
         if "language" in message: del message["language"]
+
```

Comment:
Extract this whole block in a helper function in core/nlg/\_\_init\_\_.py. Also move the two helper functions below there. Then import in both NLG classes instead of repeating in two files.